task/service.py

```diff
@@ -1,13 +1,27 @@
 import datetime
+import task.dao as task_dao
 
-def create(name: str, description: str, workspace_id: int, tags: list = None, due_date: datetime = None) -> int:
-    return -1
+def create(workspace_id: int, name: str, description: str, tags: list = None, due_date: datetime = None) -> int:
+    """Create a task."""
+    return task_dao.create(workspace_id, name, description, tags, due_date)
```

Comment:
another point of discussion for a DB/Backend sync, I'm very much in favor of a due_date field but I think the consensus we came to last semester was that all scheduling related features were stretch goals. I'm totally open to re-evaluating this as a team though!

Reply:
sounds like said conversation might need to be a whole-team meeting then